src/module/identifier/model.cpp

```
@@ -0,0 +1,365 @@
+            }
+
+            auto result = explain_infer_result(request, info);
+            callback(std::move(result));
+
+            request.set_callback([](const std::exception_ptr&) {});
+        });
+        request.start_async();
+    }
+};
+
+OpenVinoNet::OpenVinoNet() : pimpl{std::make_unique<Impl>()} {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: pimpl

(cppcoreguidelines-pro-type-member-init)